lizmap/modules/admin/templates/server_information.tpl

```diff
@@ -167,7 +171,7 @@
             {* Fixed in lizmap_server plugin 1.3.2 https://github.com/3liz/qgis-lizmap-server-plugin/commit/eb6a773ba035f877e9fa91db5ef87911a2648ee1 *}
             <th style="width:20%;">
                 {$version['name']}
-                {if $version['homepage']}
+                {if array_key_exists('homepage', $version)}
```

Comment:
Hum, now, the `homepage` is always provided, but the string can be empty I think.
```suggestion
                {if array_key_exists('homepage', $version) && $version['homepage']}
```
??

indeed, nice to check if the key exists anyway.

Reply:
Home page is undefined for me : see jelix debug bar:
 ![image](https://github.com/user-attachments/assets/9007ee41-6e68-47ce-aa1f-9ccf772dbc28)

Reply:
Ok, thanks.
So should we do both ? if the string is existing, but empty ?

Reply:
yeah that's it, need to keep both.